File: Expenses/Expenses.Model/Dto/GetCashSummaryResponseDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Expenses.Model.Dto
{
    public class GetCashSummaryResponseDto
    {
        public List<Money> HouseholdMoney { get; set; }
        public List<Money> HouseholdExpenses { get; set; }
        public List<Wallet> UserWallets { get; set; }
        public List<Money> UserExpenses { get; set; }
        public Dictionary<string, List<Money>> UserCharges { get; set; }
    }
}

File: Expenses.AndroidClient/Expenses.Repository/src/main/java/net/azurewebsites/expenses_by_klaudia/model/GetCashSummaryResponseDto.java
package net.azurewebsites.expenses_by_klaudia.model;

import java.util.HashMap;
import java.util.List;

public class GetCashSummaryResponseDto {
    public List<Money> HouseholdMoney;
    public List<Money> HouseholdExpenses;
    public List<Wallet> UserWallets;
    public List<Money> UserExpenses;
    public HashMap<String, List<Money>> UserCharges;
}
